### DISCUSSION TITLE
Hide "status" pill from element cards in CKEditor when entry type has "show status field" disabled.

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#15636] [Ideas] [answered]
The newly added "status" pill added to element cards makes things clearer for authors in many situations, but can be distracting or even confusing for simple entries in CKEditor fields, like buttons, single images etc. 

The author doesn't need to see that the CKEdItor image (embedded image entry) is enabled, especially if he doesn't have a visible way to disable it: For some of these entry types we set `show status` to `false` to declutter the edit view. Therefore, I think it would be cool if this setting would also hide the status pill from cards in CKEditor.

_Alternatively_ you could add a separate `show status in CKEditor` setting to entry types.

**Comment by brandonkelly** [comment, marked as answer]
Good suggestion. Just made this change for Craft 5.4 (bf9c66a50e18c6576b39929f11a229f7149c4663).

**Reply by brandonkelly** [reply]
Craft 5.4.0 is out with that change.
